src/CONST.ts

```diff
@@ -6484,6 +6484,7 @@ const CONST = {
             HOLD: 'hold',
             UNHOLD: 'unhold',
             DELETE: 'delete',
+            MOVE: 'move',
```

Comment:
I think that we should refer to this as change report in the code, since that's more descriptive and what we're ultimately doing 

```suggestion
            MOVE: 'changeReport',
```